src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -65,7 +65,7 @@ private static void addComplexTransformationTest(Application application) {
 			new ComplexTransformerCommand(complexTransformerCommands);
 		application.addTest("Complex transform",
 			new SelectTransformCommandOptionListener(
-				DriverFeature.getDriverManager(), complexTransformerCommand, "Transform"));
+				DriverFeature.getDriverManager(), complexTransformerCommand, "Complex"));
```

Comment:
Too bad, that details of transformations are explained inside the app (you could introduce more detailed names or logs)